eslint.config.mjs

```diff
@@ -0,0 +1,68 @@
+import { includeIgnoreFile } from '@eslint/compat';
+import { defineConfig, globalIgnores } from 'eslint/config';
+import * as path from 'node:path';
+import baseConfig from '@mui/infra/eslint';
```

Comment:
Would it make sense for these to be factory functions? So that we can pass configuration such as tsconfig locations:

```tsx
  {
    name: 'Base config',
    extends: createBaseConfig(),
    rules: {
      'import/prefer-default-export': 'off',
      // No time for this
      'react/prop-types': 'off',
      'jsx-a11y/control-has-associated-label': 'off',
      'jsx-a11y/no-autofocus': 'off',
    },
  },
```

Reply:
Yeah. Makes sense.